test/components/granules/overview.js

```diff
@@ -0,0 +1,104 @@
+'use strict';
+
+import test from 'ava';
+import Adapter from 'enzyme-adapter-react-16';
+import React from 'react';
+import { Provider } from 'react-redux';
+import { shallow, configure } from 'enzyme';
+import { GranulesOverview } from '../../../app/scripts/components/granules/overview';
+
+configure({ adapter: new Adapter() });
+
+const granules = {
+  list: {
+    meta: {
+      count: 12,
+      queriedAt: 0
+    }
+  },
+  map: {
+    'my-granule-id': {
+      data: {
+        name: 'my-name',
+        filename: 'my-filename',
+        bucket: 'my-bucket',
+        status: 'success',
+        files: [
+          {
+            fileName: 'my-name',
+            key: 'my-key-path/my-name',
+            bucket: 'my-bucket'
+          }
+        ]
+      }
+    }
+  }
+};
+
+test('Overview renders bulkAction buttons with recovery button', function (t) {
+  const dispatch = () => {};
+  const workflowOptions = [];
+  const stats = { count: 0, histogram: {}, stats: {} };
+  const location = { pathname: 'granules' };
+  const config = { recoveryPath: 'recover' };
+  const store = {
+    subscribe: () => {},
+    dispatch: dispatch,
+    getState: () => {}
+  };
+
+  const providerWrapper = shallow(
+    <Provider store={store}>
+      <GranulesOverview
+        granules = {granules}
+        stats = {stats}
+        dispatch = {dispatch}
+        workflowOptions = {workflowOptions}
+        location = {location}
+        config={config}/>
```

Comment:
So in reality, `config` comes from the store, but in this test, we mock by passing it directly as a prop? That's fine, just making sure that I understand